onnx/backend/test/cpp/driver_test.cc

```diff
@@ -160,6 +189,16 @@ class ONNXCppDriverTest
       }
       std::string serialized_model;
       model_.SerializeToString(&serialized_model);
+      std::cout << model_.graph().name() << std::endl;
```

Comment:
do you want to dump graph name every time? (maybe just for soft failures as you do in IsUnsupported).

Reply:
Dont worry, I will submit another pr to deal with this.
Right now TEST_P only output a meaningless number of testcase, I just found a way to output the model name instead.